src/BinSkim.Rules/EnableSpectreMitigations.cs

```diff
@@ -402,13 +416,21 @@ private static StringToVersionMap BuildAllowedLibraries()
         }
     }
 
+    [Flags]
+    internal enum ReportingOptions
+    {
+        None = 0x0,
+        Default = 0x1,
+        MasmModules = 0x2

```

Comment:
1) Default should be a selection of what the actual meaningful flags are, not a flag on its own (in this instance None or MasmModules)
2) The Default for this rule should be MasmModules

Reply:
Agree with (1).

Disagree with (2).  As we've discussed, the default behavior should be the one that applies to most users, and should be immediately actionable.  Until we have a complete story around how to mark MASM modules as validated, it should not be the default.

Reply:
yes, as we've discussed, the full reporting should be opt-in. can discuss further offline.